tests/delta.test.js

```diff
@@ -44,18 +44,23 @@ describe('Delta', function ( ) {
     var clientSettings = { units: 'mg/dl' };
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['clientSettings' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1417006048/issues/source?bid=2452379&fileBranchId=2903028#l44)